.gitignore

```diff
@@ -1 +1,2 @@
 _output
+bin
```

Comment:
Probably want to add a `.dockerignore` file too https://docs.docker.com/engine/reference/builder/#dockerignore-file